config/environments/production.rb

```diff
@@ -79,13 +79,13 @@
   config.action_mailer.perform_deliveries = true
   config.action_mailer.raise_delivery_errors = false
   config.action_mailer.default :charset => "utf-8"
-  
+
   config.action_mailer.smtp_settings = {
-  address: "127.0.0.1",
-  port: 25,
-  domain: 'fsektionen.se',
-  authentication: "plain",
-  enable_starttls_auto: false  
+    address: "127.0.0.1",
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.